### DISCUSSION TITLE
Can Neovim use this as the language server for C/C++?

### DISCUSSION THREAD

**Discussion by tristone13th** [microsoft/vscode-cpptools#9040] [Q&A] [answered]
I don't want to use clcs and clangd as the language server for Neovim, can Neovim use this as the language server for C/C++?

**Comment by michelleangela** [comment, marked as answer]
The C/C++ extension is published for VS Code editor and this language server is specific to this extension and can't be use on other code/text editors other than VS Code. Build system extensions for VS Code can send IntelliSense information to this extension though using this API https://github.com/Microsoft/vscode-cpptools-api.
